server/odc-service/src/main/java/com/oceanbase/odc/service/audit/AuditEventAspect.java

```diff
@@ -438,7 +457,7 @@ private AuditEvent setConnectionRelatedProperties(AuditEvent auditEvent, String
         /**
          * If sid represents static connectionId
          */
-        if (!StringUtils.contains(sid, "-")) {
+        if (!StringUtils.contains(sid, "sid:")) {
```

Comment:
I changed the '-' to 'sid:' because before V4.2.3, sid's format is like 'sid:1-1' while after V4.2.3 sid is encrypted like 'sid:eyJkYklkIjoxOSwiZHNJZCI6MSwiZnJvbSI6IkItUTk2RE1ENlItMDMzMy5sb2NhbCIsInJlYWxJZCI6IjAxODNhNGYxOGYifQ==:d:testenv', so '-' could not be a flag to distinguish connection id and dynamic sid any more.

Reply:
`DefaultConnectSessionIdGenerator#getKeyFromId`

this method may help you get connection id, datasource id and schema name

Reply:
> `DefaultConnectSessionIdGenerator#getKeyFromId`
> 
> this method may help you get connection id, datasource id and schema name

I do not need to get the connection id and schema name here. I just need to know if this is a dynamic sid or a static connection id and that's enough.